docs/how-to/hip_runtime_api/memory_management/device_memory.rst

```diff
@@ -0,0 +1,52 @@
+.. meta::
+  :description: This chapter describes the device memory of the HIP ecosystem
+                ROCm software.
+  :keywords: AMD, ROCm, HIP, device memory
+
+.. _device_memory:
+
+*******************************************************************************
+Device memory
+*******************************************************************************
+
+Device memory exists on the device, e.g. on GPUs in the video random access
+memory (VRAM), and is accessible by the kernels operating on the device. Recent
+architectures use graphics double data rate (GDDR) synchronous dynamic
+random-access memory (SDRAM) such as GDDR6, or high-bandwidth memory (HBM) such
+as HBM2e. Device memory can be allocated as global memory, constant, texture or
+surface memory.
+
+Global memory
+================================================================================
+
+Read-write storage visible to all threads on a given device. There are
```

Comment:
In this current state it's too close to the conceptual/programming model. We should add code snippets explaining how to use these different memory spaces. This does not need to be done in this PR

Reply:
Created a separate issue about this already.